exercises/tmclass_exercises/text_manipulation.py

```diff
@@ -50,10 +51,14 @@ def remove_accents(text):
     # - `unicodedata.combining(c)` returns whether `c` is a combining character
     #   (in particular accents and other diacritical marks).
     # - It is possible to assemble characters into (unicode) strings using the
-    #   `+` operator: `"abc" + "123" == "abc123"`
+    #   + operator: `"abc" + "123" == "abc123"`
+    text = unicodedata.normalize("NFD", text)
+    reconsti_text = ""
```

Comment:
Please use English for variable names to stay consistent with the rest of the file conventions.